openpgp/packet/public_key_v3.go

```diff
@@ -100,14 +100,14 @@ func (pk *PublicKeyV3) parseRSA(r io.Reader) (err error) {
 	if len(pk.n.bytes) < 8 {
 		return errors.StructuralError("v3 public key modulus is too short")
 	}
-	if len(pk.e.bytes) > 3 {
+	if len(pk.e.bytes) > 7 {
```

Comment:
What's the quantity that needs to fit in the one spare byte for 32-bit arithmetic? Is it still guaranteed to fit in one byte for 64-bit arithmetic?